src/components/asset-list/RecyclerAssetList2/WrappedCollectiblesHeader.tsx

```diff
@@ -34,7 +35,9 @@ const getMenuItemIcon = (value: NftCollectionSortCriterion) => {
 };
 
 const CollectiblesHeader = () => {
-  const { nftSort, nftSortDirection, updateNFTSort } = useNftSort();
+  const { accountAddress } = useAccountSettings();
+  const { getNftSort, updateNftSort } = useNftSortStore();
```

Comment:
Currently this will react to any changes to the `useNftSortStore`.
```suggestion
  const { getNftSort, updateNftSort } = useNftSortStore(state =>({
    getNftSort: state.getNftSort,
    updateNftSort: state.updateNftSort
  }));
```